matcher/dashboard/templates/imports/show.html

```diff
@@ -26,6 +26,9 @@
               {% endif %}
             </dd>
 
+            <dt>Effective date</dt>
+            <dd>{{ file.effective_date() }}</dd>
```

Comment:
shouldn't be a function call

Reply:
Indeed. I would blame whiskey except I’m sober. Let’s blame lack thereof. Fixed.